src/dtproject/assets/schemas/configurations.yaml.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Duckietown Dev Container Configurations",
+    "type": "object",
+    "patternProperties": {
+      ".*": {
+        "type": "object",
+        "properties": {
+          "extends": {
+            "type": "array",
+            "items": { "type": "string" },
+            "description": "A list of configurations to extend from.",
+            "uniqueItems": true
+          },
+          "plain": { "type": "boolean"}
+        }
+      }
+    }
+  }
```

Comment:
I merged this with the `containers.json` schema and renamed it `containers.yaml.json`